### DISCUSSION TITLE
Enable automatic co-host for core team

### DISCUSSION THREAD

**Discussion by whitevo** [Catalyst-Swarm/Catalyst-Beehive#232] [General]
Every now and then we run into issues where we forgot to give the host or ask for a co-host or we cant even have a host because the host cant join for various reasons.

This guide: https://support.zoom.us/hc/en-us/articles/206330935-Enabling-and-adding-a-co-host
theoretically should show how to give co-host access to other zoom accounts on the same organization.

**Comment by stephen-rowan** [comment]
I suggest we start documenting what we need to do for each TH and Saturday Swarm Session here ( https://catalyst-swarm.gitbook.io/catalyst-swarm-genesis/governance/after-town-halls )

**Comment by stephen-rowan** [comment]
Related to this Issue - Create a checklist of things to do and say when session starts #42
